tensorhive/core/managers/TensorHiveManager.py

```diff
@@ -1,55 +1,40 @@
 from threading import Thread
 from tensorhive.core.utils.Singleton import Singleton
-from tensorhive.core.managers.ThreadedManager import ThreadedManager
 from tensorhive.core.managers.InfrastructureManager import InfrastructureManager
 from tensorhive.core.managers.SSHConnectionManager import SSHConnectionManager
 from tensorhive.core.managers.ServiceManager import ServiceManager
-
-
 from tensorhive.core.services.Service import Service
 from typing import List, Dict
-
-# FIXME Remove colorama dependency from setup.py
-from colorama import Fore, Back, Style
 from tensorhive.core.utils.decorators.override import override
 from tensorhive.config import CONFIG, SSH_CONFIG, API_CONFIG
 from tensorhive.api.APIServer import APIServer
+from tensorhive.core.utils.StoppableThread import StoppableThread
+import logging
+log = logging.getLogger(__name__)
 
 
-class TensorHiveManager(ThreadedManager, metaclass=Singleton):
+class TensorHiveManager(StoppableThread, metaclass=Singleton):
     """Heart of the whole engine"""
 
     def __init__(self):
         super().__init__()
         self.infrastructure_manager = InfrastructureManager()
-
-        print('{bg_red}WARNING! You need to replace hostnames and usernames to your own in tensorhive/config.py{style_reset}'.format(bg_red=Back.RED,style_reset=Style.RESET_ALL))
-
+        log.warning('You need to replace hostnames and usernames to your own in tensorhive/config.py')
```

Comment:
This warning suggests that I forgot to configure something. Maybe it should be INFO level or show warning only if the usernames are not configured?

Reply:
Fixed: 154a6ac2df349ee2c584e6decaceb3386fa7e4f5
Good enough?